app/poros/campaign.rb

```diff
@@ -1,4 +1,4 @@
-class DndCampaign
+class Campaign
```

Comment:
Thanks for changing this to campaign, was going to do the same on my first branch.